client/hud_cl.lua

```diff
@@ -265,3 +265,18 @@ function StopAnim()
 		prop = 0
 	end
 end
+
+CreateThread(function()
+	while true do
+	Citizen.Wait(0)

```

Comment:
- Change this to `Wait(500)`. Checking if ped is in a vehicle every 0ms is unnecessary. 
- Move it to the bottom of the thread for consistency.